cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup CANOLA oil
```

Comment:
canola should have been spelled in small letters ("canola") instead of capital letters ("CANOLA")